src/ui/base-page-storyEdit.js

```diff
@@ -252,8 +262,8 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         $(pageContainer).toggleClass(hiddenEditorClass, !authoringEnabled);
     };
 
-    sjrk.storyTelling.base.page.storyEdit.submitStory = function (that, errorEvent) {
-        var form = that.container.find("form");
+    sjrk.storyTelling.base.page.storyEdit.submitStory = function (storyEditor, storyModel, errorEvent) {
+        var form = storyEditor.container.find("form");
```

Comment:
Can you use the `storyEditorForm` selector instead of `find`? Also I couldn't leave a comment there, but here are some suggestions for the ajax call. 

1. for the data property you should be able to use `formData || form.serialize()`
2. the error message could be assigned like `var errorMessage = fluid.get(jqXHR, ["responseJSON", "message"]) || "Internal server error";` You may also want to put these messages in the `strings` block of the component.
3. Could you use [DataSource](https://github.com/fluid-project/infusion/blob/master/src/framework/core/js/DataSource.js) for this?